deps/wazuh_testing/wazuh_testing/api.py

```diff
@@ -92,3 +100,42 @@ def get_security_resource_information(**kwargs):
         return response.json()['data']['affected_items'][0]
     else:
         return {}
+
+
+def get_manager_configuration(section=None, field=None):
+    """
+        Get Wazuh manager configuration response from API using GET /manager/configuration
+        https://documentation.wazuh.com/current/user-manual/api/reference.html#operation/api.controllers.manager_controller.get_configuration
+
+        Args:
+        section (str): Indicates the wazuh configuration section, for example: "active-response", "alerts"...
+        field   (str): Indicate a section child. E.g, fields for ruleset section are: decoder_dir, rule_dir, etc
+
+        Returns:
+            active configuration indicated by Wazuh API. If section and field are selected, it will return a String,
+            if not, it will return a map for the section/entire configurations with fields/sections as keys
+    """
```

Comment:
```suggestion
        Returns:
            `obj`(str or map): active configuration indicated by Wazuh API. If section and field are selected, it will return a String,
            if not, it will return a map for the section/entire configurations with fields/sections as keys
            
        Raises:
        
    """
```